api/src/main/java/com/codeforcommunity/dto/site/SiteEntry.java

```diff
@@ -0,0 +1,300 @@
+package com.codeforcommunity.dto.site;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import java.sql.Timestamp;
+
+public class SiteEntry {
+  private final Integer id;
+  private final String username;
+
+  @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "MM/dd/yyyy")
```

Comment:
🤔 You _might_ have a problem on the frontend with date parsing, for LLB we had this problem and removed this so that all timestamps were returned as an integer.

Reply:
We haven't so far but thanks for the heads up!